docs/howto/custom-template-tags.txt

```diff
@@ -784,12 +784,15 @@ sub-rendering, set the auto-escape attribute to the current context's value.
 The ``__init__`` method for the ``Context`` class takes a parameter called
 ``autoescape`` that you can use for this purpose. For example::
 
-    from django.template import Context
+    from django.template import Context, Node
 
-    def render(self, context):
-        # ...
-        new_context = Context({'var': obj}, autoescape=context.autoescape)
-        # ... Do something with new_context ...
+    class MyNode(Node):
+        ...
+        def render(self, context):
+            ...
+            new_context = Context({'var': obj}, autoescape=context.autoescape)
+            # Do something with new_context
+            ...
```

Comment:
Please revert changes in an example, we have a separate ticket-30923 to fix it and as a part of it we will remove imports.

Reply:
I left only the text change.